src/main/java/org/parabot/randoms/runewild/MacAddressFix_Runewild.java

```diff
@@ -0,0 +1,104 @@
+package org.parabot.randoms.runewild;
+
+import java.util.UUID;
+import org.parabot.api.output.Logger;
+import org.parabot.core.Context;
+import org.parabot.core.asm.ASMClassLoader;
+import org.parabot.core.reflect.RefClass;
+import org.parabot.core.reflect.RefField;
+import org.parabot.environment.randoms.Random;
+import org.parabot.environment.randoms.RandomType;
+
+/**
+ * @author EmmaStone
+ */
+public class MacAddressFix_Runewild implements Random {
```

Comment:
Rename this to MacAddressFix and remove the old class for Loco

Reply:
Done